src/bmp280/javaupm_bmp280.i

```diff
@@ -1,11 +1,20 @@
 %module javaupm_bmp280
+
+%import "../interfaces/javaupm_iTemperatureSensor.i"
+%import "../interfaces/javaupm_iHumiditySensor.i"
+%import "../interfaces/javaupm_iPressureSensor.i"
+
+
 %include "../upm.i"
+%include "cpointer.i"
+%include "typemaps.i"
+%include "arrays_java.i";
+%include "../java_buffer.i"
 
-%include "../interfaces/javaupm_iTemperatureSensor.i"
-%include "../interfaces/javaupm_iHumiditySensor.i"
-%include "../interfaces/javaupm_iPressureSensor.i"
+%typemap(javaimports) SWIGTYPE %{
+	import upm_interfaces.*;
+%}
 
-%include "bmp280_regs.h"
```

Comment:
Hi, 

Why remove this line?  By doing so you prevent the use of the symbols defined there for methods that need them like setMeasureMode() for the SWIG languages don't you?